docs/README.md

```diff
@@ -105,6 +110,31 @@ At the moment the only available external relationship is `bitwarden` with the f
 }
 ```
 
+#### Relationships with metadata
+
+A relationship can store additional information, in the `metadata` attribute. For instance, see the [contracts](https://github.com/cozy/cozy-doctypes/blob/master/docs/io.cozy.accounts.md#contracts-contracts) relationship for the accounts.
```

Comment:
we should link to docs.cozy.io not github

Reply:
Good catch :)

Reply:
Damn, I merged too fast